docs/whatsnew/3.2.rst

```diff
@@ -90,6 +90,35 @@ compute the best-fit parameters for a given frequency, as well as
 :func:`~astropy.timeseries.LombScargle.design_matrix` and
 :func:`~astropy.timeseries.LombScargle.offset` to inspect the model further.
 
+
+.. _whatsnew-3.2-codata:
+
+New SI/CODATA 2018 Constants
+============================
+
+The new redefinition of the SI system and its base units came into force on May
+20, 2019.  Accompanying that redefinition was a new set of physical constants
```

Comment:
Please change the middle-endian. Whether to stick with ISO 8601 (according to the astropy style guide to docs) or a more narrative "20 May 2019",  I don't mind.